runtime/binding-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/asyncapi/internal/model/resolver/AsyncapiCorrelationIdResolver.java

```diff
@@ -24,12 +25,13 @@
 public final class AsyncapiCorrelationIdResolver extends AbstractAsyncapiResolver<AsyncapiCorrelationId>
 {
     public AsyncapiCorrelationIdResolver(
-        Asyncapi model)
+        Asyncapi model,
+        Set<String> unresolved)
     {
         super(
             Optional.ofNullable(model.components)
                 .map(c -> c.correlationIds)
                 .orElseGet(Map::of),
-            Pattern.compile("#/components/correlationIds/(.+)"));
+            Pattern.compile("#/components/correlationIds/(.+)"), unresolved);
```

Comment:
```suggestion
            Pattern.compile("#/components/correlationIds/(.+)"),
            unresolved);
```

Same readability formatting feedback for other constructors too.